specs/InfoBadge/InfoBadge-spec.md

```diff
@@ -0,0 +1,526 @@
+
+The dot InfoBadge should be used for general scenarios in which you want to guide the user’s focus towards the InfoBadge – i.e. indicating new content or updates are available.
+
+### Icon InfoBadge
+The icon InfoBadge is a 16px by 16px ellipsis that holds an icon inside of it. InfoBadge has an `IconSource` property which provides flexibility for the types of supported icons.  
+
+The icon InfoBadge should be used to send a quick message along with getting the user’s attention – i.e. alerting the user that something non-blocking in the application has gone wrong, an extra important update is available, or that something specific in the app is currently enabled (like a countdown timer going). 
+
+Note that if you'd like to use BitmapIcon for the `IconSource` of your InfoBadge, you are responsible for ensuring that the BitmapIcon fits inside of the InfoBadge (either by changing the size of the icon, or changing the size of the InfoBadge). 
+ 
+### Numeric InfoBadge 
+The numeric InfoBadge is also a 16px by 16px ellipsis that holds a number inside of it. The width of this ellipsis will automatically expand as the number grows to multiple digits, with a smooth animation. 
+
+The numeric InfoBadge should be used for showing that there are a specific number of items that need attention – i.e. new emails/messages.
+
+## Preset InfoBadge styles
+To help support the most common scenarios in which InfoBadges are used, WinUI provides built-in, preset InfoBadge styles. While you can customize your InfoBadge to use any color/icon/number combination that you desire, these built-in presets are a quick option to make sure that your InfoBadge is compliant with accessibility guidelines and is proportional in terms of icon sizing (if you’re using an icon InfoBadge). 
+
+### List of style presets
+The following InfoBadge style presets each use one of the following colors for the background fill of the InfoBadge:
+
+![Colors available through InfoBadge style presets](images/color-breakdown.PNG)
+
+| Style Name                      | Color                   | Icon |
+|---------------------------------|-------------------------|------|
+| AttentionIconInfoBadgeStyle     | Attention/Informational | ![Asterisk InfoBadge icon](images/asterisk-badge-icon.png)|
+| InformationalIconInfoBadgeStyle | Attention/Informational |![Informational InfoBadge icon](images/informational-badge-icon.png)      |
+| SuccessIconInfoBadgeStyle       | Success                 |![Success InfoBadge icon](images/checkmark-badge-icon.png)      |
+| CautionIconInfoBadgeStyle       | Caution                 |![Caution InfoBadge icon](images/caution-badge-icon.png)      |
+| CriticalIconInfoBadgeStyle      | Critical                | ![Error InfoBadge icon](images/error-badge-icon.png)     |
+| AttentionDotInfoBadgeStyle      | Attention/Informational | None |
+| InformationalDotInfoBadgeStyle  | Attention/Informational | None |
```

Comment:
I assume these are just the tiny dot but recolored based on the banner image above of the color spectrum?

Since Attention/Informational are both Blue, what's the actual difference between these two styles?

Reply:
Yep, these would just be dot-type InfoBadges with specific background colors. 

You're right about the informational and attention styles - I don't think they'd actually have different style definitions. I think I originally just separated them for semantics (I thought someone might be looking for `InformationalDotInfoBadgeStyle` if they'd seen `InformationalIconInfoBadgeStyle` and be lost when it didn't exist). But, if they have the same definitions it's probably just unnecessary clutter.